paper.tex

```diff
@@ -88,34 +89,34 @@ \section{Motivation for central RSE units}
 \end{quotation}
 
 The quote above is the shortest possible summary of this chapter: most if not all motivation to provide RSE services stems from the goal of improving research.
-Tasks RSEs perform include training, \eg{} to improve the often low-quality code developed by beginners~\autocite{Ostlund2023}, consultation services, \eg{} regarding frameworks or algorithm selection, and the development on existing or of new software.
+Tasks RSEs perform include training, \eg{} to improve the often low-quality code developed by beginners~\autocite{Ostlund2023}, consultation services, \eg{} regarding frameworks or algorithm selection, and the development of existing or new software.
 For an overview of typical tasks of RSEs and the competencies required, see~\autocite{goth_foundational_competencies_2024}, especially section\ 4.4:\ “RSE tasks and responsibilities”.
 
 \subsection{Pooling: a necessary ingredient}
-As undoubtedly beneficial RSEs are for research, the main focus of the present paper lies on central RSE teams.
-Their main advantages all stem from the pooling of resources.
-There are at least three pooling dimensions for research institutions to benefit from: funding, diverse knowledge, and support contacts.
-The first, pooling of \textbf{funding}, allows organisations to invest in human resources through long-term expert RSEs.
-A central RSE team on long-term contracts will act as a knowledge hub due to their experience in and support of several disciplines as well as established contacts within the organisation.
+The main focus of this paper lies on central RSE teams since the benefits of RSEs to research are described elsewhere [REF].
```

Comment:
```suggestion
The main focus of this paper lies on central RSE teams since the benefits of RSEs to research are described elsewhere **TODO: [REF]**.
```